### ISSUE TITLE
Incorrect formating PDF

### ISSUE THREAD

**Issue by jjoloka** [treeio/treeio#142]
Incorrectly formatted PDF document. In the document falls menu and other page content.

**Comment by tovmeod** [comment]
Can you elaborate on that so I can recriate the issue?

**Comment by funkotron** [comment]
The PDF view is supposed to use the print css https://github.com/treeio/treeio/blob/2.0/static/print.css to remove extraneous elements. Please check the same view is produced when running `File > Print`, if this is not the case then the PDF view needs to be updated to use print.css.

**Comment by jjoloka** [comment]
No, not File > Print. Incorrectly formating PDF report, which generates wkhtmltopdf, for example - export report to pdf in reports module.

**Comment by funkotron** [comment]
Can you try File > Print and see if the issue affects that as well?
